packages/core/cli/attest.ts

```diff
@@ -76,7 +76,7 @@ const attest = async (args: any) => {
         args.contract,
         provider
     )
-    const attestingFee = await unirepContract.attestingFee()
+    const attestingFee = (await unirepContract.config()).attestingFee
```

Comment:
Could destructure here

Reply:
This same type syntax is used in lots of other places. It's fine to leave it as is